### PR TITLE
add consumer

### PR DESCRIPTION
**Pull request by ekaterinaryzhenkova**

### CODE REVIEW

File: src/NewsService.Broker/GetNewsConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      _mapper = mapper;
+    }
+
+    private async Task<object> GetNewsAsync(IGetNewsRequest request)
+    {
+      List<NewsData> News = new();
+
+      foreach(var obj in request.NewsIds)
+      {
+        DbNews dbNews = await _repository.GetAsync(obj);
```
Comment (MariyaSin) [inline src/NewsService.Broker/GetNewsConsumer.cs:33]: it's bad way to receive news on 1st. You must reduce the number of queries to the database to a minimum. Use other repo method or create new to get news by news ids

File: src/NewsService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+    [AutoInjectRequest(typeof(IGetDepartmentsRequest))]
+    public string GetDepartmentsEndpoint { get; set; }
+
+    [AutoInjectRequest(typeof(ICreateDepartmentEntityRequest))]
+    public string CreateDepartmentEntityEndpoint { get; set; }
+
+    [AutoInjectRequest(typeof(ICheckUsersExistence))]
+    public string CheckUsersExistenceEndpoint { get; set; }
+
+   // [AutoInjectRequest(typeof(IGetNewsRequest))]
```
Comment (irlirla) [inline src/NewsService.Models.Dto/Configuration/RabbitMqConfig.cs:30]: do you need this dublicate?

File: src/NewsService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
+using LT.DigitalOffice.Models.Broker.Requests.Department;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using LT.DigitalOffice.Models.Broker.Requests.News;
+using LT.DigitalOffice.Models.Broker.Requests.User;
+
+namespace LT.DigitalOffice.NewsService.Models.Dto.Configuration
+{
+  public class RabbitMqConfig : BaseRabbitMqConfig
+  {
+    public string GetNewsDataEndpoint { get; set; }
```
Comment (irlirla) [inline src/NewsService.Models.Dto/Configuration/RabbitMqConfig.cs:13]: Any AutoInjectRequest?

Comment (actimellka) [reply src/NewsService.Models.Dto/Configuration/RabbitMqConfig.cs:13]: no need here

**Review by MariyaSin — APPROVED** [review]

**Review by irlirla — APPROVED** [review]

**Review by VladSpitsin — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**